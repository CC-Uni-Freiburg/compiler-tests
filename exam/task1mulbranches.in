17
2551
157219
