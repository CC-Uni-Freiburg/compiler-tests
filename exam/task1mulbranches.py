# Author: Alexander Pfefferle
#in=17
#in=2551
#in=157219
#golden=32319990146228877828701
a = input_int()
b = 19
a = a * b
print(a)
a = input_int()
b = 78362
a = b * a
print(a)
c = input_int()
b = 183679
a = b * c
print(a)
