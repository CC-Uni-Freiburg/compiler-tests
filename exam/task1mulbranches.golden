32319990146228877828701
